src/components/App/CustomAppBar.js

```diff
@@ -47,6 +48,8 @@ const CustomAppBar = (props) => {
 
                 <div style={{ flexGrow: '1' }} />
 
+                <ImportStudyList />
+
```

Comment:
Still not sure if the AppBar is the best place for this, but for now I think it should probably be the 3rd item in the list after `Load`